src/Aspire.Hosting/ApplicationModel/CommandsConfigurationExtensions.cs

```diff
@@ -12,9 +12,9 @@ internal static class CommandsConfigurationExtensions
     internal const string StopType = "stop";
     internal const string RestartType = "restart";
 
-    internal static IResourceBuilder<T> WithLifeCycleCommands<T>(this IResourceBuilder<T> builder) where T : IResource
+    internal static void AddLifeCycleCommands(this IResource resource)
```

Comment:
nit: This type is called CommandsConfigurationExtensions. Since this is no longer an extension method type, consider a different name.

Reply:
It’s still an extension method

Reply:
Oh yeah. But not on builder.

Anyway, it doesn't matter.